modules/client/test/integration/client-addressList/methods.test.ts

```diff
@@ -184,7 +184,7 @@ describe("Client Address List", () => {
 
       const params: ICanVoteParams = {
         address: "0x1234567890123456789012345678901234567890",
-        proposalId: "0x1234567890123456789012345678901234567890_0x1",
+        proposalId: "0x1234567890123456789012345678901234567890",
```

Comment:
I believe this should be a `BigInt` if we are not using the subgraph definition for proposalID

Reply:
mmm here I disagree
Identifiers that can be copied+pasted by the user should be hex strings. Internal binary values should be exchanged in a language-native way, but I don't think this is the case